zhaquirks/tuya/__init__.py

```diff
@@ -1609,6 +1609,15 @@ def handle_get_data(self, command: TuyaCommand) -> foundation.Status:
                 dp_error = True
                 # return foundation.Status.UNSUPPORTED_ATTRIBUTE
 
+        _LOGGER.debug(
+            "[0x%04x:%s:0x%04x] Received value %s " "for attribute 0x%04x",
```

Comment:
This log was in `‎‎TuyaManufClusterAttributes` but missing in `TuyaNewManufCluster`. I can move it to a separate PR if needed.

Reply:
I think this should be fine.